react/styles.css

```diff
@@ -0,0 +1,5 @@
+.imageElement {
+}
+
+.imageElementLink {
+}
```

Comment:
what is that for?

Reply:
Oh! You mean the empty CSS classes?
I find it to be pretty strange too hahaha but that is necessary to enable customization using CSS overrides on Store Framework. Basically, the user could use this `.imageElementLink` class name to write his own implementation of this class, customizing our component through that

Reply:
Wow, this is weird. But understandable hahah!